loader.js

```diff
@@ -153,6 +153,12 @@ module.exports = function(source) {
     module.exports = doc;
     `
 
+    // Make copy before named exports are added to the doc
+    // see https://github.com/apollographql/graphql-tag/issues/168
+    outputCode += `
+    var docCopy = Object.assign({}, doc);
```

Comment:
this is partially my fault for not establishing what our compatibility matrix is, but this almost certain will break for many browsers. is it possible to use an alternative method here to assign / clone this object?

Reply:
Sure. The only properties on GraphQL AST's are `kind`, `definitions`, and `loc`, right? Would you like me to change the other instances of `Object.assign()` while I'm at it?

Reply:
ah, yes, i didnt notice that we had those in there as well. if its not too much trouble, that would be great! otherwise im happy to merge this and we can address the compatibility issue afterwards (since its already an issue)